app/models/stat.rb

```diff
@@ -0,0 +1,21 @@
+class Stat < ActiveRecord::Base
+
+  def self.sports
+    games = Game.all
+    sports = []
+    games.each do |game|
+      sports << game.sport
+    end
+
+    sports.to_a.map!(&:serializable_hash)
+    array = []
+    sports.each do |sport|
+      array << sport["name"]
+    end
+
+    @@sport_count = Hash.new(0)
+    array.each{|key| @@sport_count[key] += 1}
```

Comment:
Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.